kiliautoml/utils/helpers.py

```diff
@@ -149,6 +150,30 @@ def get_assets(
     return assets
 
 
+def get_label(asset: AssetT, strategy: LabelMergeStrategyT):
+    TYPE_ORDER = {
+        v: i for i, v in enumerate(["REVIEW", "DEFAULT", "PREDICTION", "INFERENCE", "AUTOSAVE"])
+    }
+
+    def last_order(json_response):
+        return (-TYPE_ORDER[json_response["labelType"]], json_response["createdAt"])
+
+    def first_order(json_response):
+        return (TYPE_ORDER[json_response["labelType"]], json_response["createdAt"])
+
+    labels = asset["labels"]
+    if len(labels) > 0:
+        if strategy == "first":
```

Comment:
being pedantic, but this can be even more simplified:
```
def last_order(json_response):
        return (-TYPE_ORDER[json_response["labelType"]], json_response["createdAt"])

def first_order(json_response):
        return (-TYPE_ORDER[json_response["labelType"]], -json_response["createdAt"])

labels = asset["labels"]
    if len(labels) > 0:
        max_key = first_order if strategy == "first" else last_order
        return labels.max(key=max_key)
...

```

Reply:
du coup je suis parti sur :
def last_order(json_response):
    return (
        TYPE_ORDER[json_response["labelType"]],
        -datetime.strptime(json_response["createdAt"], "%Y-%m-%dT%H:%M:%S.%fZ").timestamp(),
    )
et
    if len(labels) > 0:
        key = first_order if strategy == "first" else last_order
        labels.sort(key=key)
        return labels[0]